### ISSUE TITLE
Training-Validation split

### ISSUE THREAD

**Issue by BedirYilmaz** [leeyeehoo/CSRNet-pytorch#11]
Hello

Thank you for sharing such beautiful work.

May I ask which experimental setup you have used to achieve the results stated in the paper?

I saw that you have two different setups in your json files. I would be glad if I could find out which one is the official.

Did you use validation? If yes, how many of the training images have you reserved for validation set?

**Comment by leeyeehoo** [comment]
I'll update the repo soon... Sorry for the delay. Besides what you mean the two different setups?

**Comment by BedirYilmaz** [comment]
There is 

*part_A_train_with_val.json* and *part_A_train.json*.

In latter, you have he paths of 300 images right? 

But in first file you have 240 Images ehich makes me think that yoi have reserved 60 inages for validation during training. In first glance, these two looked like they belong to two different setups to me. Am I wrong?
